107/src/main/java/org/ehcache/jsr107/internal/JsrCacheLoaderWriter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.jsr107.internal;
+
+import org.ehcache.spi.loaderwriter.BulkCacheLoadingException;
+import org.ehcache.spi.loaderwriter.CacheLoaderWriter;
+
+import java.util.Map;
+
+/**
+ * Extension of the CacheLoaderWriter interface for doing tricks in 107 world.
+ */
+public interface JsrCacheLoaderWriter<K, V> extends CacheLoaderWriter<K, V> {
```

Comment:
`Eh107CacheLoaderWriter` already exists (and is probably badly named ...), but you are right, should go with `Jsr107CacheLoaderWriter`.